data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -210,6 +234,14 @@ def generate_queries(client,
     return sandbox_queries + clean_queries
 
 
+def create_and_get_sandbox_table_name(table, data_stage=None):
+    """
+    Return formatted sandbox table name.
+    """
+    base_name = f"{'_'.join(ISSUE_NUMBERS).lower().replace('-', '_')}_{table}"
+    return sb.get_sandbox_table_name(data_stage, base_name)
```

Comment:
I thought this function was creating sandbox tables but it looks like this is not doing much except concatenating. Can the function name be changed and maybe simplified to:
```suggestion
    return f`{data_stage}_{base_name}`
```